FindSTM32G491RE.cmake

```diff
@@ -0,0 +1,146 @@
+#[=======================================================================[.rst:
+FindSTM32G491RE
```

Comment:
Kind of a quibble but this BSP does support more than the specific g491re part; it handles all the g4s. Could be called FindSTM32G4xx?